src/main/java/pl/poznan/put/cs/scenariochecker/transformations/ScenarioHelper.java

```diff
@@ -14,4 +16,15 @@ private ScenarioHelper() {
     public static boolean isSpecialStep(Step step) {
         return step.getContent().startsWith("IF") || step.getContent().startsWith("FOR EACH");
     }
-}
+
+    /**
+     * @param step, list of actors
+     * @return true when step is step without Actor's
+     */
+    public static boolean isNoActorStep(Step step, List<String> actors) {
```

Comment:
Changing this method to ```isActorStep``` instead of ```isNoActorStep``` would improve readability significantly, don't you think?